akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -282,6 +306,26 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
     Source.wrap(source).via(this).toMat(sink)(Keep.both).run()
   }
 
+  /**
+   * Converts this Flow to a [[RunnableGraph]] that materializes to a Reactive Streams [[org.reactivestreams.Processor]]
+   * which implements the operations encapsulated by this Flow. Every materialization results in a new Processor
+   * instance, i.e. the returned [[RunnableGraph]] is reusable.
+   *
+   * @return A [[RunnableGraph]] that materializes to a Processor when run() is called on it.
+   */
+  def toProcessor: RunnableGraph[Processor[In @uncheckedVariance, Out @uncheckedVariance]] = {
+    Source.subscriber[In].via(this).toMat(Sink.publisher[Out])(Keep.both[Subscriber[In], Publisher[Out]])
+      .mapMaterializedValue {
+        case (sub, pub) ⇒ new Processor[In, Out] {
+          override def onError(t: Throwable): Unit = sub.onError(t)
+          override def onSubscribe(s: Subscription): Unit = sub.onSubscribe(s)
+          override def onComplete(): Unit = sub.onComplete()
+          override def onNext(t: In): Unit = sub.onNext(t)
+          override def subscribe(s: Subscriber[_ >: Out]): Unit = pub.subscribe(s)
+        }
+      }
```

Comment:
That is not possible, but there is an analogue, i.e. you can make a Flow that if joined with another Flow materializes to a Processor. To be honest, that would be the 100% consistent version:
 - Add a Sink to a Source to make it a Publisher
 - Add a Source to a Sink to make it a Subscriber
 - Join a Flow with a Flow to make it a Processor

... but I am afraid that in this case it is also the most confusing solution.